src/oatpp/parser/json/mapping/Serializer.cpp

```diff
@@ -243,7 +243,17 @@ void Serializer::serializeObject(Serializer* serializer,
       (first) ? first = false : stream->writeSimple(",", 1);
       serializeString(stream, field->name, std::strlen(field->name), serializer->m_config->escapeFlags);
       stream->writeSimple(":", 1);
+      auto streamFloatFormat = stream->floatFormat;
+      if (config->floatStringFormat != OATPP_FLOAT_STRING_FORMAT) {
+          stream->floatFormat = config->floatStringFormat;
+      }
+      if (field->info.format != OATPP_FLOAT_STRING_FORMAT) {
+          stream->floatFormat = field->info.format;
+      }
```

Comment:
Since we don't store `floatFormat` in the stream, we don't need this.

**Instead** - we will use the proper stream overloaded method during floatSerialization